stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -262,8 +265,8 @@ def _landing_page(
         base_url: str,
         conformance_classes: List[str],
         extension_schemas: List[str],
-    ) -> stac_types.LandingPage:
-        landing_page = stac_types.LandingPage(
+    ) -> Dict[str, Any]:
```

Comment:
Why is this method returning a Dict instead of `response_model.LandingPage` ?

Reply:
This is an internal method and we later modify the dictionary further. Instead of adding toggles between the two response types throughout the code, it is easier to default to a `dict` here and do a final cast when returning the response. Returning a `stac.LandingPage` would also work if that adds more clarity